pkgs/build-support/build-fhs-userenv-bubblewrap/default.nix

```diff
@@ -16,6 +16,47 @@ args @ {
 , unshareUts ? true
 , unshareCgroup ? true
 , dieWithParent ? true
+, etcFiles ? [
```

Comment:
This is not really a solution since when people overwrite this argument, they need to repeat all existing entries otherwise things break.